README.md

```diff
@@ -3,7 +3,7 @@
 
 # faster-fifo
 
-Faster alternative to Python's standard multiprocessing.Queue (IPC FIFO queue). Up to 30x faster in some configurations.
+Faster alternative to Python's standard multiprocessing.Queue (IPC FIFO queue). Up to 30x faster in some configurations. (**requires Python 3.6 or 3.7**)
```

Comment:
Should we say just "requires Python 3.6 or newer"?

Reply:
Done